hello-world/README.md

```diff
@@ -0,0 +1,26 @@
+# Conclave Hello World
+
+This is a simple app using the Conclave API. It is licensed under the Apache 2 license, and therefore you may 
+copy/paste it to act as the basis of your own commercial or open source apps.
+
+## How to run
+
+Start the host on a Linux system, which will build the enclave and host:
```

Comment:
You can actually build this on non-Linux systems, as described in the docsite. In fact there is no need to duplicate this information from the docsite. This sample is the docsite's tutorial, just have this README reference the correct page on the docsite.